src/mme-app/sec/secUtils.cpp

```diff
@@ -71,6 +73,28 @@ void SecUtils::create_kenb_key(unsigned char *kasme, unsigned char *kenb_key,
 }
 
 
+/**
+ * @brief Create Next hop value to exchange in Handover Request
+ * @param [in]kasme key
+ * @param [out]nh_key output the generated key
+ * @return void
+ */
+
+void SecUtils::create_nh_key(const unsigned char *kasme, unsigned char *nh_key,
+              const unsigned char *old_nh_key)
+{
+       uint8_t salt[35] = {0};
+       salt[0] = NEXT_HOP_FC; /*TODO : Sec algo. handle properly instead of harcoding here*/
```

Comment:
Can you please open a ticket for this. or pass me description I can open a new defect in security area.